README.rst

```diff
@@ -1,148 +1,262 @@
+================================
 FIWARE Policy Manager GE - Facts
-____________________
+================================
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
```

Comment:
Yes, it is working. It should be a temporally network problem with the provider of the badge